HW_5/main.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+import uvicorn as uvicorn
+from pydantic import BaseModel
+from fastapi import FastAPI, Request, HTTPException, Form
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+from starlette.staticfiles import StaticFiles
+
+app = FastAPI()
+app.mount('/static', StaticFiles(directory='static'), name='static')
+templates = Jinja2Templates(directory="templates")
+
+
+class User_in(BaseModel):
+    name: str
+    email: Optional[str]
+    password: str
+
+
+class User(User_in):
+    id: int
+
+
+users = [
+    User(id=1, name='user_1', email='1@gmail.ru', password='01'),
+    User(id=2, name='user_2', email='2@gmail.ru', password='012'),
+    User(id=3, name='user_3', email='3@gmail.ru', password='0123'),
+    User(id=4, name='user_4', email='4@gmail.ru', password='01234'),
+]
+
+
+@app.get('/', response_model=list[User], summary='Получить всех пользователей', tags=['Получить'])
+async def get_users():
+    return users
+
+
+@app.get('/get_html', response_class=HTMLResponse, summary='Получить шаблон', tags=['Получить'])
+async def get_html(request: Request):
+    title = "список пользователей"
+    return templates.TemplateResponse('index.html', {'request': request, 'title': title, 'users': users})
+
+
+@app.post('/get_html', summary='Добавляем нового пользователя', tags=['Добавить'])
```

Comment:
Вы наверно хотели создать `get` запрос?
```suggestion
@app.get('/get_html', summary='Добавляем нового пользователя', tags=['Добавить'])
```